src/Diff/Internal/AliasGroupListPatcher.php

```diff
@@ -37,7 +36,7 @@ public function patchAliasGroupList( AliasGroupList $groups, Diff $patch ) {
 	}
 
 	/**
-	 * @see MapPatcher
+	 * @see Diff\Patcher\MapPatcher
```

Comment:
The stuff in the `use` section [is nothing but a string map](http://maettig.com/1457600220) and not loading anything. I'm aware of that.

What you have seen are `@return`, `@param` and `@var` tags. This here is a `@see`. PHPStorm does not consider the `use` section when resolving `@see` tags (why should it?). `@see` only works with local methods, classes that are in the same namespace, or fully qualified class names.